### ISSUE TITLE
base.cto does not build properly

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/models#21]
The `./src/base.cto` file does not get processed and published. The build process report the following error:

```
11:23:36 AM: Error handling /opt/build/repo/src/cicero/base.cto
11:23:36 AM: Namespace org.accordproject.base specified in file /opt/build/repo/src/cicero/base.cto is already declared in file base.cto
```

I believe because the same namespace/cto file is baked in Cicero at:
https://github.com/accordproject/cicero/blob/master/packages/cicero-core/src/ciceromodelmanager.js

Should the `base.cto` file be removed from the repository?

**Comment by jeromesimeon** [comment]
I think this issue is due to the fact that the base model is already included/baked in the ModelManager before we deploy.

It is quite possible that this issue might go away if we get rid of the notion of Systems model in Concerto. (See https://github.com/accordproject/concerto/issues/62)

**Comment by dselman** [comment]
Namespace org.accordproject.base specified in file /Users/dselman/dev/models/src/ergo/altbase.cto is already declared in file base.cto

**Comment by jeromesimeon** [comment]
> Namespace org.accordproject.base specified in file /Users/dselman/dev/models/src/ergo/altbase.cto is already declared in file base.cto

I believe this is a different issue. Each model is built individually, but using a model manager which already contains a system model. This issue might go away once we have removed the notion of systems model.

**Comment by jeromesimeon** [comment]
> Namespace org.accordproject.base specified in file /Users/dselman/dev/models/src/ergo/altbase.cto is already declared in file base.cto

I wanted to make sure to record that Ergo is doing something really funky with the base model (hiding the transaction / event ids, in a way similar to what we now do in concerto-ui).

**Comment by jeromesimeon** [comment]
This issue will be moot once we switch to the new Concerto without systems model and the corresponding new AP base models (which will then all be user-level).

**Comment by jeromesimeon** [comment]
The separate issue of `ergo/altbase.cto` still requires a decision on how to handle relationships. See https://github.com/accordproject/concerto/issues/181

**Comment by jeromesimeon** [comment]
This issue is now resolved: fixed by switching to Concerto 1.0 without systems model and the new base Accord Project models. (Concretely the old `altbase.cto` and `base.cto` models are still around but not used anymore).
